### ISSUE TITLE
No output for tslint-sonarts despite config that should generate errors

### ISSUE THREAD

**Issue by douglas-treadwell** [SonarSource/SonarTS#740]
Using tslint-sonarts 1.7.0.

Using command line:
```
tslint --config tslint.json --project ./tsconfig.json '**/*.ts' --exclude 'node_modules/**/*.*' --format verbose
```

Also tried many other variations.  If I make a rule like:
"max-line-length": [true, 70],
Then I see plenty of regular TSLint errors.  So clearly it's analyzing the correct files.

But no matter what settings I put for 
"no-big-function": [true, 1],

Or other super low settings for cognitive complexity, etc...

No output is generated, whereas I would expect every function to fail the analysis.

**Comment by douglas-treadwell** [comment]
I'm not sure why it was a problem to use the global tslint, but if I use my local tslint then I get the expected errors.

```./node_modules/tslint/bin/tslint (all the rest the same)```

**Comment by vilchik-elena** [comment]
@douglas-treadwell you probably ran global tslint while having tslint-sonarts locally. Looks like it doesn't work this way
